extensions/jetbrains/src/util.ts

```diff
@@ -232,10 +232,10 @@ const globFromChannel = async (tool: Tool, channel: ChannelDetail) => {
   if (tool.toolName === undefined) {
     return [];
   }
-  const build = channel.history?.toolBuilds[0] ?? {};
+  const build = channel.history?.toolBuilds?.[0] ?? {};
   const directoryPatterns = build?.tool?.intelliJProperties?.directoryPatterns ?? [];
   const recentProjectsFilenames =
-    channel.history?.toolBuilds[0]?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
+    build?.tool?.intelliJProperties?.recentProjectsFilenames ?? [];
```

Comment:
line is too short to split now 🥹

Reply:
fixed, I thought that commiting through GH may be not the best idea 😄